### PR TITLE
Update zingmp3.py

### PR DESCRIPTION
**Pull request by NamPNQ**
Update for work with playlist, it same album but different url
Ex: 
http://mp3.zing.vn/playlist/Duong-Hong-Loan-apollobee/IWCAACCB.html
http://mp3.zing.vn/album/Duong-Hong-Loan-apollobee/IWCAACCB.html

### CODE REVIEW

File: youtube_dl/extractor/zingmp3.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def _real_extract(self, url):
 
 
 class ZingMp3AlbumIE(ZingMp3BaseInfoExtractor):
-    _VALID_URL = r'https?://mp3\.zing\.vn/album/(?P<slug>[^/]+)/(?P<album_id>\w+)\.html'
+    _VALID_URL = r'https?://mp3\.zing\.vn/(album|playlist)/(?P<slug>[^/]+)/(?P<album_id>\w+)\.html'
```
Comment (dstftw) [inline youtube_dl/extractor/zingmp3.py]: Do not capture group if you don't use it.

Comment (NamPNQ) [inline youtube_dl/extractor/zingmp3.py]: Are u have another way check album or playlist without use group?

Comment (dstftw) [inline youtube_dl/extractor/zingmp3.py]: Change to `(?:album|playlist)`. Also add a test.

Comment (NamPNQ) [inline youtube_dl/extractor/zingmp3.py]: Tks for feedback. I made update

2015-10-10 22:25 GMT+07:00 Sergey M. <notifications@github.com>:

> In youtube_dl/extractor/zingmp3.py
> <https://github.com/rg3/youtube-dl/pull/7123#discussion_r41698156>:
>
> > @@ -85,7 +85,7 @@ def _real_extract(self, url):
> >
> >
> >  class ZingMp3AlbumIE(ZingMp3BaseInfoExtractor):
> > -    _VALID_URL = r'https?://mp3\.zing\.vn/album/(?P<slug>[^/]+)/(?P<album_id>\w+)\.html'
> > +    _VALID_URL = r'https?://mp3\.zing\.vn/(album|playlist)/(?P<slug>[^/]+)/(?P<album_id>\w+)\.html'
>
> Change to (?:album|playlist). Also add a test.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rg3/youtube-dl/pull/7123/files#r41698156>.
>



-- 
*Trí tưởng tượng quan trọng hơn kiến thức.*
*__Albert Einstein__*
[image: photo]
*Phạm Ngọc Quang Nam*
Developer
m:0169.22.23.24.9 | e:nampnq@gmail.com | w:http://www.nampnq.cf
<https://www.facebook.com/nampnq>  <http://www.linkedin.com/in/nampnq>
<https://twitter.com/nampnq>  <https://plus.google.com/+NamPhamNgocQuang/>

**Comment by dstftw** [comment]
Cherry-picked in 97ae4d166c43b7e5068a7563bc5fbce385d86c2b, thanks.

**Pull request closed without merging**